### PR TITLE
docs(s3): Add doc on SSE-C MTA-4652

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

**Review by lavigne958 — APPROVED** [review]

File: storage/object/api-cli/enable-sse-c.mdx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+2. Run the following command to generate the base64-encoded 128-bit MD5 digest of your encryption key:
+
+    ```sh
+    export KEY_DIGEST=$(echo -n $KEY | openssl dgst -md5 -binary | base64)
+    ```
+
+3. Store your encryption key and its digest in a safe place.
+
+<Message type="warning">
```
Comment (bene2k1) [inline storage/object/api-cli/enable-sse-c.mdx:57]:
```suggestion
<Message type="important">
```

not sure "warning" exists. Guess it should be "important"

**Review by bene2k1 — APPROVED** [review]

File: storage/object/api-cli/enable-sse-c.mdx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+The encryption key and its digest are sent alongside the data to the cloud storage service. The data is then encrypted using the key provided during the write operation.
+Once the data has been copied, the encryption key is deleted. The digest of the key is saved with the object's metadata, allowing the data to be decrypted when downloaded.
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An [Object Storage bucket](/storage/object/how-to/create-a-bucket/)
+- installed and initialized the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
```
Comment (RoRoJ) [inline storage/object/api-cli/enable-sse-c.mdx:37]:
```suggestion
- Installed and initialized the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
```

File: storage/object/api-cli/enable-sse-c.mdx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+tags: object storage server side encryption sse sse-c cli s3
+dates:
+  validation: 2024-05-30
+  posted: 2024-05-30
+categories:
+  - storage
+  - object-storage
+---
+
+**S**erver-**S**ide **E**ncryption with **C**ustomer-provided keys (SSE-C) is a method of encryption provided by Scaleway Object Storage to protect your data at rest. It allows you to supply your own encryption keys to encrypt data when it is uploaded, and to decrypt data when it is accessed.
```
Comment (ofranc) [inline storage/object/api-cli/enable-sse-c.mdx:16]:
```suggestion
**S**erver-**S**ide **E**ncryption with **C**ustomer-provided keys (SSE-C) is a method of encryption provided by Scaleway Object Storage to protect your data at rest. It allows you to supply your own encryption keys to encrypt data when it is uploaded and decrypt data when accessed.
```

File: storage/object/api-cli/enable-sse-c.mdx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+2. Run the following command to generate the base64-encoded 128-bit MD5 digest of your encryption key, and export it as an environment variable:
+
+    ```sh
+    export KEY_DIGEST=$(echo -n $KEY | openssl dgst -md5 -binary | base64)
+    ```
+
+3. Store your encryption key and its digest in a safe place.
+
+<Message type="warning">
+If you lose the encryption key, you also lose the data encrypted with it, as you will not be able to perform `GET` operation on encrypted objects without the corresponding key.
```
Comment (RoRoJ) [inline storage/object/api-cli/enable-sse-c.mdx:58]:
```suggestion
If you lose the encryption key, you also lose the data encrypted with it, as you will not be able to perform `GET` operations on encrypted objects without the corresponding key.
```

File: storage/object/api-cli/enable-sse-c.mdx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
+**S**erver-**S**ide **E**ncryption with **C**ustomer-provided keys (SSE-C) is a method of encryption provided by Scaleway Object Storage to protect your data at rest. It allows you to supply your own encryption keys to encrypt data when it is uploaded, and to decrypt data when it is accessed.
+
+<Message type="note">
+Currently, Scaleway does not support other types of server-side encryption methods, such as SSE-KMS, or SSE-S3.
+</Message>
+
+## SSE-C overview
+
+Managing SSE-C using the `aws s3api` command set requires 3 elements:
```
Comment (ofranc) [inline storage/object/api-cli/enable-sse-c.mdx:24]:
```suggestion
Managing SSE-C using the `aws s3api` command set requires three elements:
```

**Review by RoRoJ — APPROVED** [review]

File: storage/object/api-cli/enable-sse-c.mdx (hunk 2 above)
Comment (ofranc) [inline storage/object/api-cli/enable-sse-c.mdx:37]:
```suggestion
- Installed and initialized the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
```

**Review by ofranc — APPROVED** [review]

**Pull request merged**